docs/src/theme/Footer/index.tsx

```diff
@@ -60,6 +60,8 @@ function Footer(): JSX.Element | null {
           backgroundColor: '#ef5552'
         }}
         cmdShortcutKey='m'
+        showPopup={false}
```

Comment:
@milanholemans just to confirm. I am not actually sure the impact of this change. Could you add a comment to this one 
I only noticed that after your changes the M icon is 
![image](https://github.com/pnp/cli-microsoft365/assets/58668583/f55d3a35-54ec-46d5-b319-d499efe76776)

currently, on our docs page it is 
![image](https://github.com/pnp/cli-microsoft365/assets/58668583/e29494ed-6401-4df5-837f-250e7cc035b0)

I don't see any other bigger changes 🤔 At least on desktop 😉

Reply:
It also disables the pop-up message from the button which is quite annoying on desktop according to me. On mobile devices the pop-up is still visible with the round button variant.

Reply:
ok thanks for confirming.
Now I actually understand what you mean 
![image](https://github.com/pnp/cli-microsoft365/assets/58668583/ade78df5-7999-41ef-a132-cf58290cb8fd)

Indeed it is annoying 😄

Reply:
I like the circle icon, but it seems to be impossible to use it on desktop without the annoying popup 🤷‍♂️